src/ontology/mondo-ingest.Makefile

```diff
@@ -275,3 +274,30 @@ mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-
 	# The $^ includes the python script, basically all paramters after the colon :
 
 lexical_matches: mappings/mondo-sources-all-lexical.sssom.tsv
+
+#############################
+###### Slurp pipeline #######
+#############################
+slurp/:
+	mkdir -p $@
+
+# TODO: Move this to Makefile. how, when I can't edit it here?
+.PHONY: component-download-mondo.owl
+component-download-mondo.owl: | $(TMPDIR)
```

Comment:
@matentzn Is this purl indeed correct?: http://purl.obolibrary.org/obo/mondo.owl . I got it from `tmp/mondo.owl`

Reply:
I'm curious. Is http://purl.obolibrary.org/obo/mondo.owl updated at the same time as the [latest release](https://github.com/monarch-initiative/mondo/releases/latest)?

Reply:
This url (http://purl.obolibrary.org/obo/mondo.owl) is now used by two goals: `tmp/mondo.owl` and `component-download-mondo.owl`. Should we make a variable for it and share that variable?

Reply:
@matentzn Does my `component-download-mondo.owl` look correct? I repurposed it from `component-download-omim.owl`

Reply:
> I'm curious. Is http://purl.obolibrary.org/obo/mondo.owl updated at the same time as the [latest release](https://github.com/monarch-initiative/mondo/releases/latest)?

Yes

> This url (http://purl.obolibrary.org/obo/mondo.owl) is now used by two goals: tmp/mondo.owl and component-download-mondo.owl. Should we make a variable for it and share that variable?

Dont worry I may refactor this. I would prefer if only `tmp/mondo.owl` existed.

Reply:
Thanks for these responses.

Sounds good to me. I can guess why. I'll leave the refactoring to you!